src/Extensions/CommentsExtension.php

```diff
@@ -414,7 +444,7 @@ public function getRssLink()
      */
     public function getCommentRSSLink()
     {
-        return Controller::join_links(Director::baseURL(), 'CommentingController/rss');
```

Comment:
You can now do `Director::absoluteLink('comments/rss')` and it will automatically work with base url.